addons/mass_mailing/migrations/9.0.2.0/openupgrade_analysis.txt

```diff
@@ -1,30 +1,76 @@
 ---Fields in module 'mass_mailing'---
 mass_mailing / link.tracker             / mass_mailing_campaign_id (many2one): NEW relation: mail.mass_mailing.campaign
+# Nothing to do
+
 mass_mailing / link.tracker             / mass_mailing_id (many2one)    : NEW relation: mail.mass_mailing
+# Nothing to do
+
 mass_mailing / link.tracker.click       / mail_stat_id (many2one)       : NEW relation: mail.mail.statistics
+# Nothing to do
+
 mass_mailing / link.tracker.click       / mass_mailing_campaign_id (many2one): NEW relation: mail.mass_mailing.campaign
+# Nothing to do
+
 mass_mailing / link.tracker.click       / mass_mailing_id (many2one)    : NEW relation: mail.mass_mailing
+# Nothing to do
 mass_mailing / mail.mail.statistics     / links_click_ids (one2many)    : NEW relation: link.tracker.click
-mass_mailing / mail.mass_mailing        / active (boolean)              : NEW 
-mass_mailing / mail.mass_mailing        / keep_archives (boolean)       : NEW 
-mass_mailing / mail.mass_mailing        / schedule_date (datetime)      : NEW 
+# Nothing to do
+
+mass_mailing / mail.mass_mailing        / active (boolean)              : NEW
+# Nothing to do
+
+mass_mailing / mail.mass_mailing        / keep_archives (boolean)       : NEW
+# Nothing to do
+
+mass_mailing / mail.mass_mailing        / schedule_date (datetime)      : NEW
+# Nothing to do
+
 mass_mailing / mail.mass_mailing        / state (selection)             : selection_keys is now '['done', 'draft', 'in_queue', 'sending']' ('['done', 'draft', 'test']')
-mass_mailing / mail.mass_mailing.campaign / _inherits (False)             : NEW 
+# Done ---> 'test' is converted to 'draft'
+
+mass_mailing / mail.mass_mailing.campaign / _inherits (False)             : NEW
+# Nothing to do
+
 mass_mailing / mail.mass_mailing.campaign / campaign_id (many2one)        : NEW relation: utm.campaign, required: required, req_default: function
+# Nothing to do
+
 mass_mailing / mail.mass_mailing.campaign / category_ids (many2many)      : DEL relation: mail.mass_mailing.category
+# Nothing to do
```

Comment:
You should preserve this column and rename the table for any possible inherited module that uses it.